frontend/appflowy_flutter/lib/core/helpers/url_validator.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:dartz/dartz.dart';
+
+Either<Uri, FormatException> parseValidUrl(String url) {
```

Comment:
Should be something like
```
Either<UriFailure, Uri>
```
Doesn't make sense to catch an exception just to return or rethrow it.